Classes/Controller/AbstractController.php

```diff
@@ -249,7 +249,9 @@ public function injectGetPostVarAdapterFactory(Tx_PtExtlist_Domain_StateAdapter_
     public function initializeAction()
     {
         parent::initializeAction();
-        $this->initFeUser();
+        if (!TYPO3_MODE === 'BE') {
```

Comment:
Shouldn't be that `if (TYPO3_MODE !== 'BE')` as TYPO3_MODE is not a boolean which could be negated.